### PR TITLE
Adding reliability logs

### PR DESCRIPTION
**Pull request by kaadhina**
Adding logs to capture reliability issues.
Task 568109

### CODE REVIEW

**Comment by prawalagarwal** [comment]
Looks fine.

**Comment by nigurr** [comment]
Looks fine.

File: Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1 [hunk 1]
```diff
@@ -79,30 +81,39 @@ $personalAccessToken = Get-PersonalAccessToken $vssEndpoint
 
 if (!$personalAccessToken)
 {
-    Write-Host "##vso[task.logissue type=error;code=001002;]"
+    Write-Host "##vso[task.logissue type=error;code=Unable to generate Personal Access Token for the user;TaskName=DTA]"
```
Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: I guess there's message property also in the task lib. check once

**Comment by vimegh** [comment]
_**-VerifyTestMachinesAreInUseScriptLocation $null**_

Is there any specific reason for setting this as $null, or is this a typo.

This is causing verify script location as null, leading to failure of Test Agent Deployment on new machine.

earlier this used to be 
_**-VerifyTestMachinesAreInUseScriptLocation $verifyTestMachinesAreInUse**_

@kaadhina  Please check.

**Pull request merged**